WooCommerce/Classes/ViewRelated/Products/PriceInputViewModel.swift

```diff
@@ -86,20 +86,29 @@ final class PriceInputViewModel {
     /// Returns the footer text to be displayed with information about the current bulk price and how many products will be updated.
     ///
     var footerText: String {
-        let numberOfProducts = productListViewModel.selectedProductsCount
+        var footerData: [String] = []
+
+        if productListViewModel.variableProductsCount > 0 {
+            footerData.append(Localization.variationsWarning)
+        }
+
+        let numberOfProducts = productListViewModel.selectedProductsCount - productListViewModel.variableProductsCount
         let numberOfProductsText = String.pluralize(numberOfProducts,
-                                                      singular: Localization.productsNumberSingularFooter,
-                                                      plural: Localization.productsNumberPluralFooter)
+                                                    singular: Localization.productsNumberSingularFooter,
+                                                    plural: Localization.productsNumberPluralFooter)
+        footerData.append(numberOfProductsText)
 
         switch productListViewModel.commonPriceForSelectedProducts {
         case .none:
-            return [Localization.currentPriceNoneFooter, numberOfProductsText].joined(separator: " ")
+            footerData.insert(Localization.currentPriceNoneFooter, at: 0)
         case .mixed:
-            return [Localization.currentPriceMixedFooter, numberOfProductsText].joined(separator: " ")
+            footerData.insert(Localization.currentPriceMixedFooter, at: 0)
         case let .value(price):
             let currentPriceText = String.localizedStringWithFormat(Localization.currentPriceFooter, formatPriceString(price))
-            return [currentPriceText, numberOfProductsText].joined(separator: " ")
+            footerData.insert(currentPriceText, at: 0)
         }
```

Comment:
nit: For readability, what do you think of putting this `switch` first and then appending the variations warning and number of products? That way the strings are added to `footerData` in the same order they'll appear in the returned string.

Reply:
Good idea! Updated in 4f9d55c.